hazelcast/src/test/java/com/hazelcast/internal/ascii/HTTPCommunicator.java

```diff
@@ -74,8 +74,13 @@ public String getClusterInfo() throws IOException {
     }
 
     public String getClusterHealth() throws IOException {
+        String suffix ="";
+        return getClusterHealthWithSuffix(suffix);
+    }
+
+    public String getClusterHealthWithSuffix(String suffix) throws IOException {
```

Comment:
I think I can see your point here, and it is a very interesting one. I am not 100% sure I get the why  so my understanding is the following:
 _When we see this method's name, it is not directly apparent that this method leads to failure and furthermore we cannot see how and when it fails. Which could confuse future users._

Am I correct?